tools/SampleAnalyzer/Commons/Base/Configuration.cpp

```diff
@@ -40,8 +40,8 @@ using namespace MA5;
 // Initializing static data members
 // -----------------------------------------------------------------------------
 // DO NOT TOUCH THESE LINES
-const std::string Configuration::sampleanalyzer_version_ = "1.10.1";
-const std::string Configuration::sampleanalyzer_date_    = "2022/01/15";
+const std::string Configuration::sampleanalyzer_version_ = "1.10.2";
```

Comment:
@BFuks shall we leave version bump to for the end of the PR? Each PR can correspond a version bump thats why I'm creating a new PR per update. Substrucutre module will come after all the other updates that we have been merging. Please lets keep them separated. They can all merge in the main together with appropriate version bumps otherwise we will constantly face with conflicts.

Reply:
Yep... I didn't one to ```git add``` those... It is a mistake :D